sample_config/server_multiple_port.conf

```diff
@@ -36,7 +36,8 @@ server {
 
     location /cgi/ {
         root /var/www/html/site_80/scripts;
-        cgi_extension .py;
+        extension .py;
+        cgi_path /usr/bin/python3;
```

Comment:
Why do we remove `cgi_extension` and add `extension`?
Can we choose config option names as we desire and why you decided to change it?